integration/proxy/proxy_test.go

```diff
@@ -1307,11 +1307,13 @@ func TestALPNProxyRootLeafAuthDial(t *testing.T) {
 	require.NoError(t, err)
 
 	ctx := context.Background()
-	proxyClient, err := client.ConnectToProxy(context.Background())
+
+	clusterClient, err := client.ConnectToCluster(ctx)
 	require.NoError(t, err)
 
+	rootAuthClient, err := clusterClient.ConnectToCluster(ctx, "root.example.com")
+
 	// Dial root auth service.
-	rootAuthClient, err := proxyClient.ConnectToAuthServiceThroughALPNSNIProxy(ctx, "root.example.com", "")
 	require.NoError(t, err)
```

Comment:
Move error handling closer, move comment?

```suggestion
	// Dial root auth service.
	rootAuthClient, err := clusterClient.ConnectToCluster(ctx, "root.example.com")
	require.NoError(t, err)
```